src/mlpack/methods/preprocess/preprocess_scale_main.cpp

```diff
@@ -0,0 +1,174 @@
+/**
+ * @file preprocess_scale_main.cpp
+ * @author jeffin sam
+ *
+ * A CLI executable to split a dataset.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#include <mlpack/prereqs.hpp>
+#include <mlpack/core/util/mlpack_main.hpp>
+#include <mlpack/core/math/random.hpp>
+#include <mlpack/core/util/cli.hpp>
+#include <mlpack/core/data/scaler_methods/max_abs_scaler.hpp>
+#include <mlpack/core/data/scaler_methods/mean_normalization.hpp>
+#include <mlpack/core/data/scaler_methods/min_max_scaler.hpp>
+#include <mlpack/core/data/scaler_methods/whitening.hpp>
+#include <mlpack/core/data/scaler_methods/standard_scaler.hpp>
+
+PROGRAM_INFO("Scale Data",
+    // Short description.
+    "A utility to perform feature scaling.",
+    // Long description.
+    "This utility takes a dataset and performs feature scaling using one of "
+    "the five scaler methods namely: max_abs_scaler, mean_normalization, "
+    "min_max_scaler ,standard_scaler and whitening. The function takes a "
```

Comment:
This is pedantic, but for consistency can you put the types of scaling in `''`s?  i.e. `'max_abs_scaler', 'mean_normalization', ...`

Reply:
Changed :)